code/integration-tests/local-integration-tests/src/prelude.rs

```diff
@@ -34,7 +34,7 @@ pub type USDT = XcmCurrency<statemine_runtime::Runtime, 1984, 6, 6>;
 #[allow(non_camel_case_types)]
 pub type xUSDT = XcmCurrency<this_runtime::Runtime, 1984, 12, 6>;
 
-pub type STABLE = XcmCurrency<statemine_runtime::Runtime, 666, 12, 3>;
+pub type STABLE = XcmCurrency<statemine_runtime::Runtime, 666, 3, 3>;
```

Comment:
we have all assets normalized by 12. please clarify why you are changing it here?

Reply:
Was an unintentional change - I will revert it

Reply:
Fixed in https://github.com/ComposableFi/composable/commit/acb66bb5779798366179eaf3a69191ad84d80873, thanks :+1: